src/util/token.ts

```diff
@@ -0,0 +1,26 @@
+import { Issuer } from "../types";
+import { contractInstance } from "../common/smartContract/contractInstance";
+
+const getContractInstance = (issuers: Issuer[]) => {
+  return issuers
+    .map(issuer => (issuer.tokenRegistry ? contractInstance({ contractAddress: issuer.tokenRegistry }) : null))
+    .filter(instance => !instance);
+};
+
+export const getOwnerOf = async (_tokenId: string, issuers: Issuer[]): Promise<string> => {
+  const contractInstances = getContractInstance(issuers);
+
+  const result = await Promise.all(contractInstances.map(instance => instance.ownerOf(_tokenId)));
+  const owner = result.find(o => o);
+  return owner;
+};
+
+export const transferTokenOwnership = async (_from: string, _to: string, _tokenId: string, issuers: Issuer[]) => {
```

Comment:
Seemed like you are trying to support multiple token contract? I think there was a discussion where if its a token we dont support more than on contract....

@rjchow ?

Reply:
yea we aren't supporting more than one

Reply:
If that's the case lets do a validation early and throw when we see one funky token with more than one token backing it.

Reply:
put a check before.